lambdas/src/apis/contracts/DAOCache.ts

```diff
@@ -0,0 +1,31 @@
+import { ServerMetadata } from "decentraland-katalyst-commons/ServerMetadata";
+import { TimeRefreshedDataHolder } from "./TimeRefreshedDataHolder";
+import { DAOContractClient } from "decentraland-katalyst-commons/DAOClient";
+import ms from "ms";
+import { DAOListClient } from "./DAOListsClient";
+
+const REFRESH_TIME: number = ms('15m')
```

Comment:
I think we could event set this higher, like `30m`. These values will rarely change

Reply:
You're right, I'll set it to 30m.